src/index.html

```diff
@@ -6,8 +6,48 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div class="card" data-qa="card">
+      <a href="#" class="card__link">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          >
```

Comment:
```suggestion
        <img
          class="card__image"
          src="./images/imac.jpeg"
          alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
        >
```

small formatting issue, also check please formatting in all places, issue still exists